### PR TITLE
{math}[gompi/2020b,iimpi/2020b] SCOTCH v6.1.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.9-gompi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name = 'SCOTCH'
+version = '6.0.9'
```
Comment (boegel) [inline easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.9-gompi-2020b.eb:2]: should be updated to 6.1.0, see https://gforge.inria.fr/forum/forum.php?forum_id=11546...

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3626.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a59c936b4ca9f652ce6828e2229c0e27 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3115.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/da8e9824a3f98c8bfc655efe9b6f0627 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2613.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/fb6d48a62496a55b11baa536e2b1bbda for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/3b6f79e140573b264babb623804978fe for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/5b31b069d70c362b0cf040bda0784a69 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**